datm/cime_config/stream_definition_datm.xml

```diff
@@ -33,13 +33,14 @@
   CLMGSWP3v1            = Run with the CLM GSWP3 V1 forcing (force CLM)
   CLMNLDAS2             = Run with the CLM NLDAS2 regional forcing valid from 1980 to 2018 (force CLM)
   1PT                   = Run with supplied single point data (force CLM)
-  CORE2_NYF             = CORE2 normal year forcing (for forcing POP and CICE)
-  CORE2_IAF             = CORE2 intra-annual year forcing (for forcing POP and CICE)
-  CORE_IAF_JRA          = JRA55 intra-annual year forcing (for forcing POP and CICE)
-  CORE_IAF_JRA_1p4_2018 = JRA55 intra-annual year forcing, v1.4, through 2018 (for forcing POP and CICE)
-  CORE_RYF8485_JRA      = JRA55 repeat year forcing, v1.3, 1984-1985 (for forcing POP and CICE)
-  CORE_RYF9091_JRA      = JRA55 repeat year forcing, v1.3, 1990-1991 (for forcing POP and CICE)
-  CORE_RYF0304_JRA      = JRA55 repeat year forcing, v1.3, 2003-2004 (for forcing POP and CICE)
+  CORE2_NYF             = CORE2 normal year forcing (for forcing MOM6 and CICE)
+  CORE2_IAF             = CORE2 intra-annual year forcing (for forcing MOM6 and CICE)
+  CORE_IAF_JRA          = JRA55 intra-annual year forcing (for forcing MOM6 and CICE)
+  CORE_IAF_JRA_1p4_2018 = JRA55 intra-annual year forcing, v1.4, through 2018 (for forcing MOM6 and CICE)
+  CORE_IAF_JRA_1p5_2023 = JRA55 intra-annual year forcing, v1.5, through 2023 (for forcing MOM6 and CICE)
+  CORE_RYF8485_JRA      = JRA55 repeat year forcing, v1.3, 1984-1985 (for forcing MOM6 and CICE)
+  CORE_RYF9091_JRA      = JRA55 repeat year forcing, v1.3, 1990-1991 (for forcing MOM6 and CICE)
+  CORE_RYF0304_JRA      = JRA55 repeat year forcing, v1.3, 2003-2004 (for forcing MOM6 and CICE)
```

Comment:
Is there a reason to have the `CORE_` prefix here? It's different than the `CORE2_` prefix used for CORE forcing, but it's similar enough to feel like maybe it's not necessary?

Reply:
I can't remember the exact reason but it was a desired prefix at the time when we first introduced JRA forcing (i.e., it was intentional).